getting_started/workflow/export/exporting_for_ios.rst

```diff
@@ -36,42 +36,42 @@ After you click **Export Project**, there are still two important options left:
 
 .. image:: img/ios_export_file.png
 
-.. note:: I chose the name **exported_xcode_project_name**, for this tutorial, 
-          but you will use your project's name. When you see **exported_xcode_project_name** 
+.. note:: This tutorial uses **exported_xcode_project_name**, but you will use your
+          project's name. When you see **exported_xcode_project_name** 
           in the following steps, replace it with the name you used instead. 
 
-When the export completes, my folder looks like this:
+When the export completes, the output folder looks like this:
```

Comment:
Suggestion: `the output folder should look like this`

I don't know if that would follow the grammar rules we're going for, but this feels a bit more like how it'd be worded in a tutorial.